tests/bundle/DependencyInjection/Cofniguration/Parser/SiteAccessAware/SuggestionParserTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Search\DependencyInjection\Cofniguration\Parser\SiteAccessAware;
```

Comment:
Hi,
There is a typo in parent directory and namespace:
```suggestion
namespace Ibexa\Tests\Bundle\Search\DependencyInjection\Configuration\Parser\SiteAccessAware;
```